recipes/twilio/meta.yaml

```diff
@@ -0,0 +1,213 @@
+{% set name = "twilio" %}
+{% set version = "6.32.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e78a2006b9449fb9fad5050537e0998c181c7d3a62eaa9eed434e59dbaf58324
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pyjwt >=1.4.2
+    - python
+    - pytz
+    - requests >=2.0.0
+    - six
+  run:
+    - pyjwt >=1.4.2
+    - python
+    - pytz
+    - requests >=2.0.0
+    - six
+
+test:
+  imports:
+    - twilio
+    - twilio.base
+    - twilio.http
+    - twilio.jwt
+    - twilio.jwt.access_token
+    - twilio.jwt.client
```

Comment:
It is not necessary to import all the modules and submodules. Testing just the basic modules should be fine. But if you want to you can let them here, you will just have more work when the modules e submodules changed from version to version

Reply:
Good point, hadn't thought of that. I used what the `skeleton` generated as is, which came from the `twilio` tests module, I think. Removed a lot of them in the latest commit to leave out primary submodules.